.eslintrc.json

```diff
@@ -30,18 +30,12 @@
     {
       "files": ["*.ts", "*.tsx"],
       "extends": ["plugin:@nx/typescript", "prettier"],
-      "rules": {
-        "@typescript-eslint/no-extra-semi": "error",
-        "no-extra-semi": "off"
```

Comment:
These rules were dropped by `@typescript-eslint` b/c they are more style than linting, and prettier covers them.